src/page/blog/draft-page.jsx

```diff
@@ -13,12 +32,12 @@ export default function Drafts() {
     <>
       <div className="w-full max-auto max-w-2xl h-40 left-60 relative">
         {data.posts.map((post, idx) => {
-          const { title, author } = post;
+          const { title, author, createdAt } = post;
           return (
             <Draft
```

Comment:
I am unable to edit drafts on draft page but can edit published articles. Not sure if this is just an issue on my end.